marklogic-data-hub-central/ui/e2e/cypress/e2e/explore/exploreUnmerge.cy.tsx

```diff
@@ -238,28 +233,75 @@ describe("Test '/Explore' graph right panel", () => {
     browsePage.getUnmergeIcon().first().scrollIntoView().should("be.visible");
-    cy.log("** verify compare values modal when clicking unmerge icon **");
     browsePage.getUnmergeIcon().first().click();
     compareValuesModal.getModal().should("be.visible");
     compareValuesModal.getUnmergeButton().should("be.visible");
 
     cy.log("** cancel button closes modal **");
     compareValuesModal.getCancelButton().click();
     compareValuesModal.getModal().should("not.exist");
+  });
 
-    cy.log("** reopen modal and submit unmerge **");
+  it("Unmerge record and validate include unmerged documents checkbox", () => {
+    cy.log("** Reopen modal and submit unmerge **");
     browsePage.getUnmergeIcon().first().scrollIntoView().click();
     compareValuesModal.getModal().should("be.visible");
+    compareValuesModal.getInclusionCheckbox().click();
     compareValuesModal.getUnmergeButton().click();
     compareValuesModal.confirmationYes().click();
     compareValuesModal.getModal().should("not.exist");
 
-    cy.log("** confirm merged record is unmerged **");
+    browsePage.getUnmergeIcon().should("not.exist");
+
+    graphExplore.getRunTile().click();
+    cy.intercept("GET", "/api/jobs/**").as("runResponse");
+
+    graphExplore.getPersonJSONacordeon().click();
+    graphExplore.getRunButtonMatchPerson().click();
+    cy.wait("@runResponse");
+    cy.wait(4000);
+    graphExplore.getCloseModalMatchPerson().click();
+    graphExplore.getRunButtonMergePerson().click();
+    cy.wait("@runResponse");
+    cy.wait(4000);
+    graphExplore.getCloseModalMergePerson().click({force: true});
+
+    toolbar.getExploreToolbarIcon().click();
+    browsePage.clickTableView();
+    cy.log("** Unmerge icon should be visible because of inclusion checkbox**");
+    browsePage.getUnmergeIcon().should("be.visible");
+  });
+
+  it("Unmerge record and validate document is not merged again", () => {
+    cy.log("** Reopen modal and submit unmerge **");
+    browsePage.getUnmergeIcon().first().scrollIntoView().click();
+    compareValuesModal.getModal().should("be.visible");
+    compareValuesModal.getUnmergeButton().click();
+    compareValuesModal.confirmationYes().click();
+    compareValuesModal.getModal().should("not.exist");
+
+    browsePage.getUnmergeIcon().should("not.exist");
+
+    graphExplore.getRunTile().click();
+    cy.intercept("GET", "/api/jobs/**").as("runResponse");
+
+    graphExplore.getPersonJSONacordeon().click();
+    graphExplore.getRunButtonMatchPerson().click();
+    cy.wait("@runResponse");
+    cy.wait(4000);
+    graphExplore.getCloseModalMatchPerson().click();
+    graphExplore.getRunButtonMergePerson().click();
+    cy.wait("@runResponse");
+    cy.wait(4000);
+    graphExplore.getCloseModalMergePerson().click({force: true});
+    toolbar.getExploreToolbarIcon().click();
+    browsePage.clickTableView();
+    cy.log("** Unmerge icon should not be visible because of inclusion checkbox**");
```

Comment:
@aniamy Can we also add validation where we re-run the merge step and the documents get merged without any issue.

Reply:
I'll fix the log since it was copy and paste. But the test above validates with the checkbox checked. And this test validates that if the checkbox is unchecked the documents get merged without issues and there is no unmerge option.
Do you mean running merge step again and validate there was nothing be merged?